packages/sdk/src/protocols/filter/index.ts

```diff
@@ -54,6 +54,9 @@ class FilterSDK extends BaseProtocolSDK implements IFilterSDK {
 
           await subscription.processIncomingMessage(wakuMessage, peerIdStr);
         },
+        async (error: Error) => {
```

Comment:
I think this is not needed

Reply:
yes, that is a good log to have which is already there so this error handler as a parameter can be removed all together